src/integrationTest/java/uk/gov/hmcts/darts/cases/controller/CaseControllerTest.java

```diff
@@ -123,7 +136,7 @@ void setupData() {
 
     @Test
     void casesGetEndpoint() throws Exception {
-
+        setupExternalDarPcUserForCourthouse(null);
```

Comment:
I'm finding it hard to follow why we pass in null here for this private method - it looks like you could do that inside that other method if you need to pass null through to the stub?